api/hono-trpc.ts

```
@@ -31,8 +31,9 @@ app.use(
   '/trpc/*',
   trpcServer({
     router: appRouter,
-    createContext: async (opts: FetchCreateContextFnOptions) => {
-      return await createContext({ req: opts.req });
+    createContext: async (opts: FetchCreateContextFnOptions, c) => {
```

ESLint

[error] 'c' is defined but never used.

(@typescript-eslint/no-unused-vars)